app/src/js/modules/liveeditor.js

```diff
@@ -122,11 +128,22 @@
                                     win.getSelection().getRangeAt(0).insertNode(doc.createTextNode(content));
                                 }
                             }
-                        }).on('keypress', function (e) {
-                            return e.which != 13;
-                        }).on('focus blur', function (e) {
-                            $(this).html($(this).text());
                         });
+
+                        if(fieldType == 'textarea') {
+                            $(this).on('keypress', function (e) {
+                                if(e.which == 13) {
+                                    e.preventDefault();
+                                    doc.execCommand('insertHTML', false, '<br><br>');
+                                }
+                            });
+                        } else {
+                            $(this).on('keypress', function (e) {
+                                return e.which != 13;
+                            }).on('focus blur', function (e) {
+                                $(this).html($(this).text());
```

Comment:
Yeah I completely agree with the handling of <br> characters being different, just concerned about other HTML sneaking in!

It's pretty shocking that there's no simple way to do plaintext at present, given that `contenteditable` has been around since IE5.